packages/docs-ui/README.md

```diff
@@ -16,7 +16,8 @@ npm install @braid-design-system/docs-ui
 
 ### `LinkableHeading`
 
-A heading wrapped in a link to a hash that is generated based on the heading content, enabling deep links within documentation pages.
+A heading wrapped in a link to a hash that is generated based on the heading content, enabling deep links within
+documentation pages.
```

Comment:
Why the line break here?

Reply:
Ahh my bad, thought I got it stopped when I disabled prettier, idk what I did.